packages/palette/src/elements/AutocompleteInput/AutocompleteInput.tsx

```diff
@@ -219,6 +226,12 @@ export const AutocompleteInput = <T extends AutocompleteInputOptionType>({
     onChange: handleFocusChange,
   })
 
+  const handleClose = () => {
+    dispatch({ type: "CLOSE" })
+    reset()
+    onClose?.()
```

Comment:
**question**: do we need to call `onClose` here? As far as I remember, if the user selects one of the options, then `onClose` is not called 🤔

Reply:
You do, because it's a callback, like if I pass some callback into this component to be called when it's closed, then I want that callback to be called *every* time it's closed, right?